io.github.xplshn.Openarena.yaml

```diff
@@ -0,0 +1,74 @@
+app-id: io.github.xplshn.Openarena
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+# CHANGED: Let it look up the command via $PATH
+command: launch
+modules:
+  # CHANGED: Module name isn't displayed to users and convention says lowercase
+  - name: openarena
+    buildsystem: simple
+    sources:
+      # CHANGED: Pull direct from the upstream source
+      - type: git
+        url: "https://github.com/ec-/Quake3e"
```

Comment:
Shouldn't you use https://github.com/OpenArena/engine here?

Reply:
That's outdated and has plenty of bugs on Linux, the idea behind making a new FlatPak is to replace the old one that has problems

Reply:
Have you tried working together with upstream to update the engine?

Reply:
> Have you tried working together with upstream to update the engine?

No because its based on IoQuake3, and IoQuake3 has poor performance, I suggested they add their modifications to another engine like Quake3e or Spearmint engine